editor/README.md

```diff
@@ -19,15 +19,16 @@ Everything requires Python 3.
 The build needs:
 * built ord-schema python code;
 * the `protoc` protobuf compiler;
-* the protobuf runtime libraries for Javascript; and
-* the Closure Library for Javascript.
+* the protobuf runtime libraries for Javascript;
+* the Closure Library for Javascript; and
+* Node.js and npm.
 
 Serving depends on:
 * the Flask python web framework.
 
 To build the ord-schema python code, follow the instructions [here](https://github.com/Open-Reaction-Database/ord-schema/blob/main/README.md).
 
-The editor currently requires HEAD protobuf (to get experimental "optional" declarations) for the protobuf compiler and Javascript runtime libraries.
+For the protobuf compiler and Javascript runtime libraries, the editor currently requires HEAD protobuf (to get experimental "optional" declarations).
```

Comment:
Thanks for cleaning this up!

No need to fix this now, but this file a good example of a change that is unrelated to the rest of the PR and would have been good to send as a separate one.

Reply:
Yup! My bad.